contracts/V1/StateChannelDiamondProxy/DisputeManagerFacet.sol

```diff
@@ -8,7 +8,8 @@ import "./Errors.sol";
 contract DisputeManagerFacet is StateChannelCommon {
 
     function createDispute(
-        Dispute memory dispute
+        Dispute memory dispute,
+        bytes calldata disputerSignature
```

Comment:
This signature is not needed since dispute creation checks msg.sender == dispute.disputer so the creator of the dispute is implicitly counted towards the threshold, so N-1 signatures are enough.  If it's convenient to include it, it's fine since it's not expensive

Reply:
done